Ch04/numbercrunch.cpp

```diff
@@ -0,0 +1,28 @@
+#include "../std_lib_facilities.h"
+
+double sum(const vector<double>& v) {
```

Comment:
Should consider creating `stat.h` to keep all these statistics function library.